src/components/widgets/TableWidget/TableWidget.tsx

```diff
@@ -349,6 +348,12 @@ export const TableWidget: FunctionComponent<TableWidgetProps> = props => {
         props.onSelectCell(recordId, props.meta.name, fieldKey)
     }
 
+    const fieldMeta = React.useMemo(
+        () => (item: WidgetListField) =>
+            item.type === FieldType.multivalue ? ({ ...item, type: FieldType.multivalueHover } as WidgetListField) : item,
+        []
```

Comment:
This memoization won't work; you declared a function `(item)  => ({ ...item, type })` and memoized it, and the function will always be the same, but you do not assigning a function to the prop - you are _calling_ a function and then  assigning its _result_ to the prop, and result of this function is new object on each call => each render => rerenders.

Try like this:
```ts
const fieldMeta = React.useMemo(() => {
return item.type === FieldType.multivalue ? ({ ...item, type: FieldType.multivalueHover  }) : item
, [item]}
```

Reply:
@Dergash @artkzlv What do you think about removing field type conversion in 2.0.0 in favor of correctly built widget meta?
> I guess table widget shouldn't convert field type. Type of field should be resolved on meta data layer. If widget type is List then field type is multivalueHover. Otherwise multivalue.

Let's add corresponding TODO if there are no objections.

Reply:
No objections